src/plugins/inputs/cycle.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
```

Comment:
I disagree, on the basis that `/cycle` is a long-standing name for the function (and is even a builtin on some networks, e.g. [SwiftIRC](https://wiki.swiftirc.net/wiki/Other_commands#Cycle.2C_Hop)). "rejoin" is usually attached to rejoining channels upon being kicked.

Input from other maintainers (@xPaw, @YaManicKill?) will be the deciding factor here, though. If there's consensus to canonically call this `rejoin`, I won't fight it. But it is my strongly held belief that calling it `rejoin` instead of `cycle` is more or less breaking from tradition for the sake of doing so.